Task 5 - Completed Front end for both Forgot My Password and Reset my Password pages

SPTLAbron opened the pull request:
I completed the front end pages for both the Forgot my Password page and Reset my Password page

EDIT: Frontend generally looks good and matches Figma design but does not look good when zooming in a certain amount

SPTLAbron committed:
completed front end to both pages

RickyLi15 reviewed (commented):
Hey guys, overall your pages look good. There are just some typos, if you can compare them with the figma page that would be great. Also, the designers decided to go with a dark green color instead of blue. Please change that as well.

SPTLAbron committed:
task 5 - fixed small spelling and color issues

yieb reviewed (changes requested):
It would be great if you could add more detail to the PR description and update the PR title to be more descriptive

Also it would be good if you could follow the example files in the repo and add your reset password .tsx + .css file in their own subdirectory under pages and similarly make another subdirectory for the files for the ForgotPassword page files

SPTLAbron committed:
made changes to file locations

yieb commented on jaba/src/App.tsx:
```diff
@@ -1,9 +1,11 @@
 import ExamplePage from "./pages/ExamplePage/ExamplePage";
+import ForgotPassword from "./pages/ForgotPasswordFrontEnd";
```
You put your files in subdirectories under pages/ like I asked but you also need to update the import paths in the App.tsx file so errors don't occur on build

yieb commented on jaba/src/pages/ForgotPassword/ForgotPasswordFrontEnd.module.css:
```diff
@@ -0,0 +1,86 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap');
```
Can you see if you can rebase your branch with main and use the custom font files that were added to the repo instead of importing with the url?

yieb commented on jaba/src/pages/ForgotPassword/ForgotPasswordFrontEnd.module.css:
Can you rename the files without the "FrontEnd" endings? Those are a bit unnecessary

yieb reviewed (commented):
![image](https://github.com/Hack4Impact-UMD/jefferson-area-board-for-aging/assets/52669318/3eb322e8-9d8b-4fce-b1d5-4a022dad5dde)

- The Reset Password page is missing the little eye icons in the input boxes. 
- There should also be a graphic on the right side of the page that is missing on both the Reset Password and Forgot Password pages

yieb commented on jaba/src/pages/ResetPasswordFrontEnd.module.css:
```diff
@@ -0,0 +1,88 @@
+    font-family: 'DM Serif Text', serif;
+}
+
+.login_section {
```
Remove these css styles (.login_section, .login_section_context, a) - they are unused in this particular page

yieb commented on jaba/src/pages/ResetPasswordFrontEnd.tsx:
```diff
@@ -0,0 +1,27 @@
+
+function ResetPassword() {
+    return (
+      <div className={classes.content}>
```
What is this content class? It is unused right now

SPTLAbron committed:
made all requested changes

SPTLAbron commented:
I made all the stated changes. I used online SVG's for seeing password or hiding password parts.

yieb commented on jaba/src/pages/ForgotPassword/ForgotPassword.module.css:
You did not rebase and switched to using the custom fonts that are in the main branch

yieb commented on jaba/src/pages/ForgotPassword/ForgotPassword.tsx:
```diff
@@ -0,0 +1,30 @@
+import classes from './ForgotPassword.module.css';
+import { ReactComponent as Logo } from '../image_6.svg';
```
Can you import the image with a more descriptive name

yieb commented on jaba/src/pages/ForgotPassword/ForgotPassword.tsx:
```diff
@@ -0,0 +1,30 @@
+import classes from './ForgotPassword.module.css';
```
Can you update to use the syntax that Ricky messaged? Use "styles" instead of "classes"

yieb commented on jaba/src/pages/ResetPassword/ResetPassword.tsx:
```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+
+function ResetPassword() {
+  const [isVisible1, setIsVisible1] = useState(true);
```
Could we use more descriptive react state names?

yieb commented on jaba/src/pages/image_6.svg:
Can you update the name of this image to be a bit more descriptive in some way? Additionally, can you move this to the the assets directory?

yieb commented:
![image](https://github.com/Hack4Impact-UMD/jefferson-area-board-for-aging/assets/52669318/4f0a4738-3cc5-40f8-9d6a-88867b8b15b5)
Can you double check some of your styling? Some issue but for example, when I see the page, the input box where you type the email has overly rounded corners. I see you used em in your styling, see if maybe using px like in the Figma will fix it. 
Also, the text "Send Reset Link" on the button seems a bit small and some of the font weights seem a bit off.

SPTLAbron committed:
made final changes

SPTLAbron commented:
I removed the imports for the fonts, renamed the image, moved it into the assets folder, and changed the images for each page.

yieb committed:
Merge branch 'main' into 5_login_frontend

yieb committed:
Reorder and edit CSS styling

- Remove some unused CSS styling
- Rearrange CSS style classes to match more with order of classes in tsx
  file

yieb committed:
Update alt text, remove unused CSS class, edit SVG

yieb committed:
Clean up CSS for ForgotPassword page

SPTLAbron committed:
changed eye icons location

SPTLAbron committed:
finished changes to eye icons

SPTLAbron committed:
changes style classes to camelCase

yieb committed:
Readjust font size and margin/padding values

yieb committed:
Change icons to import from react-unicons

yieb committed:
Fix eye icon centering in input boxes

yieb committed:
Remove eye icon images from assets folder

yieb committed:
Rename React states and CSS class for more clarity

yieb committed:
Add clarifying comment in CSS file

yieb committed:
Merge branch 'main' into 5_login_frontend
